cmd/aws-iam-authenticator/root.go

```diff
@@ -88,23 +88,25 @@ func initConfig() {
 func getConfig() (config.Config, error) {
 
 	cfg := config.Config{
-		PartitionID:                       viper.GetString("server.partition"),
-		ClusterID:                         viper.GetString("clusterID"),
-		ServerEC2DescribeInstancesRoleARN: viper.GetString("server.ec2DescribeInstancesRoleARN"),
-		HostPort:                          viper.GetInt("server.port"),
-		Hostname:                          viper.GetString("server.hostname"),
-		GenerateKubeconfigPath:            viper.GetString("server.generateKubeconfig"),
-		KubeconfigPregenerated:            viper.GetBool("server.kubeconfigPregenerated"),
-		StateDir:                          viper.GetString("server.stateDir"),
-		Address:                           viper.GetString("server.address"),
-		Kubeconfig:                        viper.GetString("server.kubeconfig"),
-		Master:                            viper.GetString("server.master"),
-		BackendMode:                       viper.GetStringSlice("server.backendMode"),
-		EC2DescribeInstancesQps:           viper.GetInt("server.ec2DescribeInstancesQps"),
-		EC2DescribeInstancesBurst:         viper.GetInt("server.ec2DescribeInstancesBurst"),
-		ScrubbedAWSAccounts:               viper.GetStringSlice("server.scrubbedAccounts"),
-		DynamicFilePath:                   viper.GetString("server.dynamicfilepath"),
-		DynamicFileUserIDStrict:           viper.GetBool("server.dynamicfileUserIDStrict"),
+		PartitionID:                          viper.GetString("server.partition"),
+		ClusterID:                            viper.GetString("clusterID"),
+		ServerEC2DescribeInstancesRoleARN:    viper.GetString("server.ec2DescribeInstancesRoleARN"),
+		HostPort:                             viper.GetInt("server.port"),
+		Hostname:                             viper.GetString("server.hostname"),
+		GenerateKubeconfigPath:               viper.GetString("server.generateKubeconfig"),
+		KubeconfigPregenerated:               viper.GetBool("server.kubeconfigPregenerated"),
+		StateDir:                             viper.GetString("server.stateDir"),
+		Address:                              viper.GetString("server.address"),
+		Kubeconfig:                           viper.GetString("server.kubeconfig"),
+		Master:                               viper.GetString("server.master"),
+		BackendMode:                          viper.GetStringSlice("server.backendMode"),
+		EC2DescribeInstancesQps:              viper.GetInt("server.ec2DescribeInstancesQps"),
+		EC2DescribeInstancesBurst:            viper.GetInt("server.ec2DescribeInstancesBurst"),
+		ScrubbedAWSAccounts:                  viper.GetStringSlice("server.scrubbedAccounts"),
+		DynamicFilePath:                      viper.GetString("server.dynamicfilepath"),
```

Comment:
for readability, can you add either a comment or blank newline above the `DynamicFile` options so the value indenting is grouped?

Reply:
Mentioned offline, can you put the docstrings for each variable above the config?